src/components/Proposals/ProposalPage/OPProposalApprovalPage/OptionResultsPanel/OptionResultsPanel.tsx

```diff
@@ -57,22 +57,29 @@ export default function OptionsResultsPanel({
       {sortedOptions.map((option, index) => {
         let isApproved = false;
         const votesAmountBN = BigInt(option?.votes || 0);
-        const optionBudget = parseUnits(
-          option?.budgetTokensSpent?.toString() || "0",
-          tokens.get(proposalData.proposalSettings.budgetToken)?.decimals ?? 18
-        );
+        // Date in which optionBudget was being sent correctly to chain, all other past proposals optionBudget is in
+        // ether unit instead of wei
+        const changeDate = new Date("2024-02-21T12:00:00");
```

Comment:
considering we're building a multi-tenant application, let's define things like this globally.
this might be a good place, similar to how we set v6 upgrade block: https://github.com/voteagora/agora-next/blob/main/src/lib/contracts/contracts.ts#L34

Reply:
Done and tested, LGTM!